### DISCUSSION TITLE
Plane Maker crashing

### DISCUSSION THREAD

**Discussion by zeta976** [Stratosphere-Studios/777-300ER#54] [General]
Every time I try to open the .acf in plane maker, it will keep loading for around a minute and then the app will just crash. This doesn't happen with any other aircraft I've tried. I think this might be related to the visual model being very demanding (fps are considerably lower compared to what I get with other airliners). We should consider reducing texture quality and poly-count for several objects. 
E.g.
- Cabin seats can be reduced from 2k to 1k
- Landing gear can be reduced from 4K to 2K 
- Glass texture can be reduced from 4K to 1K or even 512x512

**Comment by remenkemi** [comment]
Which version?

**Reply by zeta976** [reply]
Latest XP12 version

**Comment by remenkemi** [comment]
I don't experience this issue but I'm keeping the discussion open as a reminder for those texture size reductions.
